utils/kata-manager.sh

```diff
@@ -269,16 +269,20 @@ containerd_installed()
 
 pre_checks()
 {
+	local only_kata="${1:-}"
```

Comment:
Maybe  use `skip-containerd` here instead of `only_kata` to make things clearer.